### PR TITLE
reformatted footer

### PR DESCRIPTION
**Pull request by futrica**

### CODE REVIEW

**Comment by meirabruno** [comment]
:+1:

**Comment by jaimerson** [comment]
:shipit:

File: source/stylesheets/all.css.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 $tracer-blue: #2c3e50;
+$base-blue: #004267;
```
Comment (mpivaa) [inline source/stylesheets/all.css.scss]: Colocar variaveis dentro de _variables.css.scss

**Comment by mpivaa** [comment]
:+1:

**Pull request merged**